changelog.d/8744.bugfix

```diff
@@ -0,0 +1 @@
+Fix a bug where appservices may be sent an excessive amount of read receipts and presence
```

Comment:
I couldn't think of a good way to summarise the bug. Improvements welcome.

Reply:
One thing to note is that we often include the Synapse version that introduced the bug in bugfix newsfiles.